worker/worker.py

```diff
@@ -80,6 +82,15 @@ def on_sigint(signal, frame):
   ALIVE = False
   raise Exception('Terminated by signal')
 
+rate = None
+
+def get_rate():
+  global rate
+  rate = requests.get('https://api.github.com/rate_limit', timeout=HTTP_TIMEOUT).json()['rate']
+  remaining = rate['remaining']
```

Comment:
Would it be useful to handle the case that the api request fails?

Reply:
The idea is to catch the exception in the main loop?

Reply:
Ok. That seems safe...